packages/runtime/native/shared/Css_types.ml

```diff
@@ -1063,20 +1062,45 @@ module AnimationPlayState = struct
     | Cascading.t
     ]
 
-  let toString x =
+  let toString (x : t) =
     match x with
     | `paused -> {js|paused|js}
     | `running -> {js|running|js}
     | #Var.t as va -> Var.toString va
     | #Cascading.t as c -> Cascading.toString c
 end
 
+module SingleTransitionProperty = struct
+  type t =
+    [ `all
+    | `ident of string
```

Comment:
Do you think is worth keeping the `string` and allow "all"?

Reply:
yeah, this feels unnecessary

Reply:
nvm, i think this is needed since we want to also have Var.t and Cascading.t

Reply:
but I wonder what's the point of cascading or auto in animation?